lib/galaxy/webapps/galaxy/controllers/shed_tool_static.py

```diff
@@ -26,17 +32,25 @@ def index(self, trans, shed, owner, repo, tool, version, image_file):
         """
         guid = '/'.join([shed, 'repos', owner, repo, tool, version])
         tool = trans.app.toolbox.get_tool(guid)
-        repo_path = tool._repository_dir
-        if 'static/images' not in image_file:
-            path = join(repo_path, 'static', 'images', image_file)
+        repo_path = os.path.abspath(tool._repository_dir)
+        asset_path = os.path.abspath(join(repo_path, image_file))
+
+        # test specified image_file path exactly first, then fail through to
+        # other locations.
+        # Might want to swap this around and check for static/images first if
```

Comment:
Yeah, large majority of images are in static/images, we should check that first.